src/realmModels/ObservationPhoto.js

```diff
@@ -30,29 +30,31 @@ class ObservationPhoto extends Realm.Object {
     return localObsPhoto;
   }
 
-  static mapPhotoForUpload( observtionID, observationPhoto ) {
+  static mapPhotoForUpload( observtionID, photo ) {
     return {
-      "photo[uuid]": observationPhoto.uuid,
       file: new FileUpload( {
-        uri: observationPhoto.photo.localFilePath,
-        name: `${observationPhoto.uuid}.jpeg`,
+        uri: photo.localFilePath,
+        name: photo.localFilePath,
         type: "image/jpeg"
       } )
     };
   }
 
-  static mapPhotoForAttachingToObs( id, observationPhoto ) {
+  static mapPhotoForAttachingToObs( observtionID, observationPhoto ) {
```

Comment:
The typo bothers me a bit.

Reply:
Agreed. I fixed it in a subsequent commit.